lib/utils.js

```diff
@@ -26,19 +31,16 @@ function createElementProps(node, index) {
         key: index,
     };
     if (node.attribs) {
-        forEach(node.attribs, function (value, key) {
-            switch (key || '') {
-                case 'style':
-                    elementProps.style = createStyleJsonFromString(node.attribs.style);
-                    break;
-                case 'class':
-                    elementProps.className = value;
-                    break;
-                default:
-                    elementProps[key] = value;
-                    break;
+        elementProps = reduce(node.attribs, function(result, value, key) {
+            key = getCamelCaseAttr(key) || key;
```

Comment:
I'd prefer not introducing the `getCamelCaseAttr` function as it only obscures what's happening and doesn't reduce duplication.